WhaleSpotting/Repositories/UserRepo.cs

```diff
@@ -71,11 +74,14 @@ public User GetByUsername(string username)
         }
     }
 
-    public List<UserResponse> ListAllUsers()
+    public List<UserLeaderboardResponse> ListOfUserLeaderboard()
```

Comment:
Rename to `GetUserLeaderboard`
```suggestion
    public List<UserLeaderboardResponse> GetUserLeaderboard()
```